src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -63,7 +64,13 @@ interface ParsedRow {
 // Helper -- Takes column header from file, converts to internal metadata key
 // If header is unrecognized, leaves it alone (useful for warnings, etc).
 function convertHeaderToMetadataKey(headerName: string): string {
-  return HEADERS_TO_METADATA_KEYS[headerName] || headerName;
+  if (headerName in HEADERS_TO_METADATA_KEYS) {
+    return HEADERS_TO_METADATA_KEYS[headerName];
+  } else if (headerName in NEXTSTRAIN_FORMAT_HEADERS_TO_METADATA_KEYS) {
+    return NEXTSTRAIN_FORMAT_HEADERS_TO_METADATA_KEYS[headerName];
```

Comment:
What is the use for this second dictionary? What are the instances where the headers differ?

Reply:
sorry i didn't do a good job at explaining, so this was to handle the case for if the users wanted to upload metadata with different headers that are default for nextstrain, we're handling this for the users so that they don't need to convert the headers to match to our system, and can just upload directly.

Reply:
does that answer your question? lol

Reply:
How will this process generalize in multi pathogen? For example, will there also be nextstrain specific templates for tb, etc?

Reply:
@mmmmmmaya -- no, the nextstrain template is universal across viruses